### DISCUSSION TITLE
how to convert lammps.dat file for suitable format in deepmd for training to build deep potential

### DISCUSSION THREAD

**Discussion by Riddhiahir** [deepmodeling/deepmd-kit#3680] [Q&A]
hello!
I'm new to deepmd-kit. Right now I'm simulating gamma-graphyne sheets with water solvation box. i have generated lammps.dat file for final configuration using VMD. now to generate deep potential i want convert lammps.dat file in compatible format. please guide me through this.

**Comment by njzjz** [comment]
You can use [dpdata](https://github.com/deepmodeling/dpdata).
